organization/mission.md

```diff
@@ -2,8 +2,20 @@
 type: Resource
 ---
 
-# Mission of the Foundation for Public Code
+# Mission & founding principles
 
-We create a viable future for cities and civic operating systems that are highly participatory and drive societal engagement. A public digital infrastructure that is inclusive, (re-)usable, adaptive, open and sustainable.
+## Mission
 
-To achieve this we maintain, manage, contribute to and grow an ecosystem of public software and policies that help governments, communities and citizens improve their cities.
+To enable public-purpose software and policy that is open and collaborative.
```

Comment:
Arnout's suggestion:
```suggestion
To make public-purpose software open and collaborative.
```

Reply:
I don't think this will be useful.

First off we don't _make_ anything, and don't have the ambition too since it would lead to disempowerment.

Secondly I think undervaluing the role of policy in the digital transformation and effectively seperating it from software is the whole reason we're in this mess, so I think for us to be able to execute on our mission it is crucial that we include policy at the highest level, and software as well.

Was there any more reasoning?